kubernetes/deployment-production.tmpl

```diff
@@ -348,3 +348,64 @@ spec:
       port: 80
       targetPort: 80
   type: NodePort
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: panoptes-production-redis
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: azurefile
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Should we deploy redis as a stateful set for consistent volume / network identifiers? 

https://schoolofdevops.github.io/ultimate-kubernetes-bootcamp/13_redis_statefulset/
https://rancher.com/blog/2019/deploying-redis-cluster
https://estl.tech/deploying-redis-with-persistence-on-google-kubernetes-engine-c1d60f70a043

Reply:
I'm not familiar with stateful sets, so to avoid any surprises I'd lean towards using a deployment for now and maybe revisiting this later (because of the urgency of getting this finished). A deployment with one replica will do what we need as far as I can see.